@l10n/ja/docs/concepts/tokens/decentralized-exchange/permissioned-domains.md

```diff
@@ -0,0 +1,39 @@
+---
+seo:
+    description: 許可型ドメインの定義と詳細
+labels:
+  - コンプライアンス
+  - 許可型ドメイン
+---
+# 許可型ドメイン
+
+許可型ドメインは、XRP Ledgerブロックチェーンのより広いエコシステム内の制御された環境です。ドメインは何もしませんが、許可型DEXや貸付プロトコルなどの機能は、ドメインを使用してアクセスを制限するため、従来の金融機関は、様々なコンプライアンスルールに従って、チェーン上でサービスを提供できます。
+
+ドメインの唯一の設定可能なルールは、[資格情報][]のセットです。将来のAmendmentは、金融機関が業務を行う地域の法律に従ってコンプライアンスを維持するために必要な制限をカバーするために、新しいおよび異なるタイプのルールを追加することができます。
```

Comment:
The common link for `資格情報` is not defined in this PR, so this link is unparsed here.

Given that we use the term "Credentials" as-is in Japanese ([for example, here](https://xrpl.org/ja/docs/concepts/decentralized-storage/credentials)), all instances of "資格情報" in this PR should probably be changed to "Credentials" for consistency.

Reply:
Since this is a concept page,changed it to “資格情報(Credentials)” and fixed the link so that it works properly.

The link is to an English page, but it will be fixed by #3193. (The path of raw-partial at the bottom of the page needs to be modified.)